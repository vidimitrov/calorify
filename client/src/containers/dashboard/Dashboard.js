import React from 'react';
import PropTypes from 'prop-types';

export const Dashboard = ({ children }) => (
  <div>{children}</div>
);

Dashboard.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.element),
    PropTypes.element,
  ]).isRequired,
};

export default Dashboard;
