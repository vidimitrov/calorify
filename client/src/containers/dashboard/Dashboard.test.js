import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Dashboard from './Dashboard';

it('Dashboard container renders correctly', () => {
  const tree = renderer.create(
    <Dashboard><div /></Dashboard>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
