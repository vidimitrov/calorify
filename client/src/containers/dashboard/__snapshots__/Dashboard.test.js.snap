// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard container renders correctly 1`] = `
<div>
  <div />
</div>
`;
